cli_reference/openshift_cli/usage-oc-kubectl.adoc

```diff
@@ -21,6 +21,45 @@ The `oc` binary offers a built-in `login` command that allows authentication and
 +
 The additional command `oc new-app`, for example, makes it easier to get new applications started using existing source code or pre-built images. Similarly, the additional command `oc new-project` makes it easier to start a project that you can switch to as your default.
 
+The {product-title} version must match between master and node hosts, excluding
+temporary mismatches during cluster upgrades. For example, in a
+{product-version} cluster, all masters must be {product-version} and all nodes
```

Comment:
These two sentences are about kubelet/API-server node skew, not `oc`.  Can we talk about node skew somewhere outside of this `oc`-focused file?  I'm not sure if/where we formally doc kubelet/API-server skew today, but the in-cluster logic is described [here][1] and depends on whether the OpenShift minor version is even or odd.

[1]: https://github.com/openshift/cluster-kube-apiserver-operator/pull/1199#issue-958481467

Reply:
I don't think we document that anywhere, so if thats true we need to add it somewhere.